{{cookiecutter.slug}}/bootstrap.py

```diff
@@ -163,6 +167,98 @@ def make_create_db_command(psql_cmd):
+
+        serve = merge_json({}, data['projects'][project]['architect']['serve']['configurations']['production'])
+        serve['browserTarget'] += f'-{code}'
+        data['projects'][project]['architect']['serve']['configurations'][code] = serve
+
+    data['projects'][project]['architect']['build']['configurations']['production']['outputPath'] = 'dist/default'
+
+    with open('frontend/angular.json', 'w') as file:
+        json.dump(data, file, indent=2)
+
+def activate_frontend():
+    framework = '{{cookiecutter.frontend}}'
+    os.rename('package.{{cookiecutter.frontend}}.json', 'package.json')
+
+    if framework == 'backbone':
+        os.rename('frontend.backbone', 'frontend')
+    elif framework == 'angular':
+        project_name = '{{cookiecutter.slug}}'.replace('_', '-')
+        Command(
+            'Creating project',
+            ['ng', 'new', project_name, '--prefix={{cookiecutter.app_prefix}}',
+                '--skipGit=true',
+                '--skipInstall=true',
+                '--style=scss']
+        )()
+        dir_util.copy_tree('frontend.angular', project_name)
+        os.rename(project_name, 'frontend')
+        Command(
+            'Set project to use Yarn',
+            ['ng', 'config', 'cli.packageManager', 'yarn'],
+            cwd="frontend"
+        )()
+        Command(
+            'Install frontend dependencies using Yarn',
+            ['yarn'],
+            cwd="frontend"
+        )()
+        create_angular_localization_settings()
+        Command(
+            'Creating localizations',
+            ['ng', 'xi18n', '--output-path', 'locale'],
+            cwd="frontend"
+        )()
+        for lang in '{{cookiecutter.localizations}}'.split(','):
+            [code, lang_name] = lang.split(':')
+            shutil.copyfile('frontend/src/locale/messages.xlf', f'frontend/src/locale/messages.{code}.xlf')
+        if '{{cookiecutter.frontend_port}}' != '4200':
+            Command(
+                'Set frontend port',
+                ['ng', 'config', 'projects.{{cookiecutter.slug | replace('_', '-')}}.architect.serve.options.port', '{{cookiecutter.frontend_port}}'],
+                cwd="frontend"
+            )()
+    else:
+        print('Unknown framework {{cookiecutter.frontend}} specified!')
+    if os.path.isfile('frontend/package.overwrite.json'):
+        with open('frontend/package.overwrite.json', 'r') as file:
+            overwrite = json.load(file)
+        with open('frontend/package.json', 'r') as file:
+            data = json.load(file)
+        with open('frontend/package.json', 'w') as file:
+            merge_json(data, overwrite)
+            json.dump(data, file, indent=2)
+        os.remove('frontend/package.overwrite.json')
+    # remove other frameworks
+    for path in glob.glob("frontend.*"):
+        shutil.rmtree(path)
+    for path in glob.glob("package.*.json"):
+        os.remove(path)
```

Comment:
I recommend returning a boolean that indicates whether `activate_frontend` was successful overall, because you may want to make some steps in the `main` conditional on this.